src/conversion/view-selection-to-model-converters.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Contains {@link engine.view.Selection view selection} to {@link engine.model.Selection model selection} conversion
+ * helper.
+ *
+ * @namespace engine.conversion.viewSelectionToModel
+ */
+
+/**
+ * Function factory, creates a callback function which converts a {@link engine.view.Selection view selection} taken
+ * from the {@link engine.view.Document#selectionChange} event and set in on the
+ * {@link engine.model.Document#selection model}.
+ *
+ * Note that because there is not view selection change dispatcher nor any other advance view selection to model
+ * conversion mechanism, this method is simple event listener.
+ *
+ *		view.document.on( 'selectionChange', convertSelectionChange( model, mapper ) );
+ *
+ * @function engine.conversion.viewSelectionToModel.convertSelectionChange
+ * @param {engine.model.Document} model Model document on which selection should be updated.
+ * @param {engine.conversion.Mapper} mapper Conversion mapper.
+ * @returns {Function} {@link engine.view.Document#selectionChange} callback function.
+ */
+export function convertSelectionChange( model, mapper ) {
+	return ( evt, data ) => {
+		model.enqueueChanges( () => {
```

Comment:
Yep. Because changed selection have to be rendered, rendering run on `changesDone`, and without `enqueueChanges` you would not have a `changesDone` event.